content/workshops/ansible_tower/exercise1.0.adoc

```diff
@@ -71,6 +71,65 @@ ansible web -m setup
 
 === Step 4:
 
+Now, let's install a deliberately out-of-date version of `subversion`, a revision control tool.  Let's first find the most recent out-of-date version of the package that is available:
+
+[source,bash] 
+----
+sudo yum list available subversion.x86_64 --showduplicates
+----
+
+You should get a number of lines of output, that look something like this (though the versions will vary):
+
+[source,bash] 
+----
+Loaded plugins: amazon-id, rhui-lb, search-disabled-repos
+Available Packages
+subversion.x86_64       1.7.14-6.el7            rhui-REGION-rhel-server-releases
+subversion.x86_64       1.7.14-7.el7_0          rhui-REGION-rhel-server-releases
+subversion.x86_64       1.7.14-7.el7_1.1        rhui-REGION-rhel-server-releases
+subversion.x86_64       1.7.14-10.el7           rhui-REGION-rhel-server-releases
+subversion.x86_64       1.7.14-11.el7_4         rhui-REGION-rhel-server-releases
+subversion.x86_64       1.7.14-14.el7           rhui-REGION-rhel-server-releases
+----
+
+Now, let's install the version mentioned in the second-to-last line:
+
+[source,bash] 
+----
+ansible web -m package -a "name=subversion-1.7.14-11.el7_4 state=present" -b
+----
+
+By the way, you can use that method, referring to a package with `<package name>-<version>` to install any version of any package.
+
+Since that's done, let's now update Subversion to the current version:
+
+[source,bash] 
+----
+ansible web -m package -a "name=subversion state=latest" -b
+----
+
+====
+[NOTE]
+Package Updating Methods
+
+There are various methods that you can use to update packages.  You saw a simple case, already, but you can also update *all* packages to current with:
+
+[source,bash] 
+----
+ansible web -m package -a "name=* state=latest" -b
+----
+
+Or, you can only install updates, to all packages, with security implications, with this:
```

Comment:
Please reword the phrase "security implications".  This is too vague and needs to be more specific.  The official docs state:

> If set to yes, and state=latest then only installs updates that have been marked security related.

Everything else looks great.  Please provide more context for the security=yes parameter and resubmit